cli/cmd/upgrade_legacy.go

```diff
@@ -2,157 +2,51 @@ package cmd
 
 import (
```

Comment:
Since this file no longer has anything to do with upgrading from legacy, we should rename this file to something more appropriate.  Either that or move the functions here into repair.go.

Reply:
I moved it to `repair` go in https://github.com/linkerd/linkerd2/pull/7309/commits/212591032d05f2deb28502d68b5297ec4619150b